app/assets/javascripts/discourse/app/components/user-menu/reviewable-user-item.js

```diff
@@ -0,0 +1,13 @@
+import UserMenuDefaultReviewableItem from "discourse/components/user-menu/default-reviewable-item";
+import I18n from "I18n";
+
+export default class UserMenuReviewableUserItem extends UserMenuDefaultReviewableItem {
+  get description() {
+    const username = this.reviewable.username;
+    return I18n.t("user_menu.reviewable.suspicious_user", { username });
```

Comment:
```suggestion
    return I18n.t("user_menu.reviewable.suspicious_user", { username: this.reviewable.username });
```

The variable is kind of redundant here.